front/src/components/list-of-group/index.js

```diff
@@ -0,0 +1,3 @@
+import GroupListPage from './GroupListPage';
+
+export default GroupListPage;
```

Comment:
Component name and name of folder in which it is stored should match

Reply:
done